cli/src/build/onboarding/android/ui/app.tsx

```
@@ -112,6 +117,47 @@ function emptyProgress(appId: string): AndroidOnboardingProgress {
+
+/**
+ * Build the picker options for `pick-build-script`. Shows ALL scripts from
+ * package.json (the user picks; we don't auto-guess), with the project-type
+ * recommendation surfaced at the top, plus escape hatches for custom commands
+ * and "skip build entirely" (raw HTML Capacitor apps).
+ */
+function buildScriptPickerOptions(scripts: Record<string, string>, recommended: string | null): BuildScriptOption[] {
+  const options: BuildScriptOption[] = []
+  const seen = new Set<string>()
+
+  if (recommended && Object.prototype.hasOwnProperty.call(scripts, recommended)) {
```

ESLint

[error] Use Object.hasOwn() instead of hasOwnProperty

(e18e/prefer-object-has-own)